src/examples/common_plane_gpu.cpp

```diff
@@ -0,0 +1,280 @@
+    }
+  }
+  // move to gpu
+  tribol::ArrayT<tribol::IndexT, 2, MSPACE> conn_2(host_conn_2);
+  std::cout << "Tribol connectivity created (" << timer.elapsedTimeInMilliSec() << " ms)" << std::endl;
+
+  std::cout << "Registering Tribol mesh data..." << std::endl;
+  timer.start();
+  constexpr tribol::IndexT mesh1_id = 0;
+  tribol::registerMesh(mesh1_id, num_contact_elems, fe_space.GetNDofs(), conn_1.data(), tribol::LINEAR_QUAD,
+    x_coords_ptr, y_coords_ptr, z_coords_ptr, MSPACE);
+  constexpr tribol::IndexT mesh2_id = 1;
+  tribol::registerMesh(mesh2_id, num_contact_elems, fe_space.GetNDofs(), conn_2.data(), tribol::LINEAR_QUAD,
+    x_coords_ptr, y_coords_ptr, z_coords_ptr, MSPACE);
+
+  constexpr tribol::RealT penalty = 5000.0;
+  tribol::setKinematicConstantPenalty(mesh1_id, penalty);
+  tribol::setKinematicConstantPenalty(mesh2_id, penalty);
+  std::cout << "Tribol mesh data registered (" << timer.elapsedTimeInMilliSec() << " ms)" << std::endl;
+
+  std::cout << "Creating and registering velocity and force..." << std::endl;
+  timer.start();
+  mfem::GridFunction velocity(&fe_space);
+  velocity = 0.0;
+  auto velocity_ptr = velocity.Read();
+  auto x_velocity_ptr = &velocity_ptr[fe_space.DofToVDof(0, 0)];
+  auto y_velocity_ptr = &velocity_ptr[fe_space.DofToVDof(0, 1)];
+  auto z_velocity_ptr = &velocity_ptr[fe_space.DofToVDof(0, 2)];
+  tribol::registerNodalVelocities(mesh1_id, x_velocity_ptr, y_velocity_ptr, z_velocity_ptr);
+  tribol::registerNodalVelocities(mesh2_id, x_velocity_ptr, y_velocity_ptr, z_velocity_ptr);
+
+  mfem::Vector force(fe_space.GetVSize());
+  force.UseDevice(true);
+  force = 0.0;
+  auto force_ptr = force.ReadWrite();
+  auto x_force_ptr = &force_ptr[fe_space.DofToVDof(0, 0)];
+  auto y_force_ptr = &force_ptr[fe_space.DofToVDof(0, 1)];
+  auto z_force_ptr = &force_ptr[fe_space.DofToVDof(0, 2)];
+  tribol::registerNodalResponse(mesh1_id, x_force_ptr, y_force_ptr, z_force_ptr);
+  tribol::registerNodalResponse(mesh2_id, x_force_ptr, y_force_ptr, z_force_ptr);
+  std::cout << "Velocity and force registered (" << timer.elapsedTimeInMilliSec() << " ms)" << std::endl;
+
+  std::cout << "Registering Tribol coupling scheme..." << std::endl;
+  timer.start();
+  constexpr tribol::IndexT cs_id = 0;
+  tribol::registerCouplingScheme(cs_id, mesh1_id, mesh2_id,
+    tribol::SURFACE_TO_SURFACE, 
+    tribol::NO_CASE, 
+    tribol::COMMON_PLANE, 
+    tribol::FRICTIONLESS,
+    tribol::PENALTY,
+    tribol::BINNING_BVH,
+    EXEC);
+
+  tribol::setPenaltyOptions(cs_id, tribol::KINEMATIC, tribol::KINEMATIC_CONSTANT);
+  std::cout << "Tribol coupling scheme registered (" << timer.elapsedTimeInMilliSec() << " ms)" << std::endl;
+
+  std::cout << "Calling Tribol update..." << std::endl;
+  timer.start();
+  int cycle = 1;
+  RealT t = 1.0;
+  RealT dt = 1.0;
+  tribol::update(cycle, t, dt);
+  std::cout << "Tribol update complete (" << timer.elapsedTimeInMilliSec() << " ms)" << std::endl;
+
+  RealT max_force = force.Max();
```

Comment:
Add comment about what this returns. Is this the maximum magnitude of a nodal force vector? Is this useful? Or do we just want to report the force.Norm1() that is below?

Reply:
It's somewhat useful for debugging (i.e. if the l1 norm isn't right), so it might be worth leaving in?